### PR TITLE
Introduce mpld3 

### LINKED ISSUE #53: Modularised plotting
Considerations:
1. Zooming/Panning/etc. on an interactive
2. A clear/standardised architecture
3. Types of plots
4. Supporting the web visualisation

### PR DESCRIPTION
**Pull request by bangxiangyong**
This is to track the progress of implementing mpld3 as a way to add matplotlib figures into the web dashboard. This will resolve #53 .

We shall finalise the implementation in the future, and merge into the develop branch.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -16,3 +16,4 @@ requests
 plotly
 torch
 torchvision
+dash-dangerously-set-inner-html
```
Comment (BjoernLudwigPTB) [inline requirements.txt:19]: Ah yeah, we should add that to the `environment.yml` as well at a later time, if we really need it.

Comment (BjoernLudwigPTB) [reply requirements.txt:19]: We should actually really add it, before considering that as resolve.

**Comment by BjoernLudwigPTB** [comment]
@bangxiangyong The [requirements.txt](https://github.com/bangxiangyong/agentMET4FOF/blob/develop/requirements.txt) and with that the [environment.yml](https://github.com/bangxiangyong/agentMET4FOF/blob/develop/environment.yml) are updated now, so you can continue on this one (or merge it, if finished).

**Comment by bangxiangyong** [comment]
With https://github.com/bangxiangyong/agentMET4FOF/pull/114 which achieves what this pull request was aimed to do, I shall close this pull now.

**Pull request closed without merging**